proxmox/datadog_checks/proxmox/check.py

```diff
@@ -54,6 +66,49 @@ def _get_vm_hostname(self, vm_id, vm_name, node):
         hostname = hostname_json.get("data", {}).get("result", {}).get("host-name", vm_name)
         return hostname
 
+    def _event_for_task(self, task, node_name):
```

Comment:
```suggestion
    def _create_dd_event_for_task(self, task, node_name):
```
nit: could be more descriptive of what the method does